tests/e2e/vsan_test.go

```diff
@@ -77,12 +82,25 @@ func (s *VsanTestSuite) TearDownTest(c *C) {
 
 var _ = Suite(&VsanTestSuite{})
 
+// This function is used to return a unique container name used for "vsan_test"
+func (s *VsanTestSuite) newCName() string {
```

Comment:
There is not much left in the method other than one line `inputparams.GetUniqueContainerName("vsan_test")` after syncing up with master code. Containers will be removed automatically hence no required of L88 `s.containerList = append(s.containerList, cname)` ... This method left out to be as inline function.